tests/lib/setup_suite.bash

```diff
@@ -5,7 +5,13 @@ load '../lib/binaries'
 
 setup_suite() {
     {
+        if [[ "${HELM_SECRETS_BACKEND}" == "custom-"* ]]; then
```

Comment:
Cloud please move the if condition under line 14?

I think, in IT tests, `HELM_SECRETS_BACKEND` is undef, but its catched on line 13 here `${HELM_SECRETS_BACKEND:-"sops"}`